content/Software and Downloads/IDE Settings/If-setup-runs-twice.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "If setup() runs twice"
```

Comment:
```suggestion
title: "If your board runs the sketch twice"
```
Technically, we're not dealing with `setup()` specifically. The sketch will just run twice and can easily enter `loop()` the first time before resetting.